src/lua/events.c

```diff
@@ -628,6 +628,10 @@ static int destroy_shortcut_event(lua_State *L)
 
   // remove the accelerator from the lua shortcuts
   dt_accel_deregister_lua(tmp);
+
+  // free temporary buffer
+  free(tmp);
+
```

Comment:
Buffer has been allocated by `strdup` a few lines above. Cannot find where it is freed.

Reply:
This works fine.  You can add a free in the routine above, after line 606, to free that tmp also.

Reply:
@wpferguson Thanks for reviewing and for pointing out the other leak. I revised my pr to fix both.

Reply:
Thanks for fixing :)